File: php/secure-public-access/securing_public_access_get_article.php
<?php
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kontent\Ai\Delivery\DeliveryClient;

$client = new DeliveryClient('KONTENT_AI_ENVIRONMENT_ID', null, 'KONTENT_AI_DELIVERY_API_KEY');

$item = $client->getItem('my_article');
?>

File: java/secure-public-access/securing_public_access_get_article.java
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient with secure access key
DeliveryClient client = new DeliveryClient(
	DeliveryOptions.builder()
		.projectId("KONTENT_AI_ENVIRONMENT_ID")
		.productionApiKey("<YOUR_API_KEY>")
		.build()
);

// Gets the latest version of an item
CompletionStage<ContentItemResponse> item = client.getItem("my_article");
